docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Explainer: what is a jumpbox or bastion host?"
+description: Defines the terms 'jumpbox' and 'bastion host'
+author: telmosampaio
+---
+
+# Explainer: what is a jumpbox?
+
+In the foundational adoption stage, you learned about using a single infrastructure-as-a-service (IaaS) virtual machine (VM). Most IaaS workloads will be composed by more than a single VM. They can encompass several VMs running different services, such as a web UI, distributed micro services, and database servers. 
+
+Different users need to access these separate VMs in a secure fashion, without you necessarly providing direct access to the VMs themselves. You can do that by using a **jumpbox**, **bastion host**, or **secure administrative host**. 
```

Comment:
Last sentence: "You can do that by using a jumpbox, also known as a bastion host or secure administration host. Then you can do away with the note below.